src/index.js

```diff
@@ -0,0 +1,160 @@
+const state = {
+    currentTempButton: null,
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    tempValue: 72,
+    tempColor: null,
+    sky: null,
+    skySelect: null,
+    gardenContent: null,
+    landscape: null,
+    cityNameInput: null,
+    headerCityName: null,
+    lat: null,
+    lon: null,
+    cityName: 'Seattle',
+};
+
+const handleTempColorAndGarden = (tempValue) => {
+  if (tempValue >= 80) {
+    state.tempColor = 'red'; 
+    state.landscape = '🌵 🏜 🐍 🦂 🏝️ 🦀 🐚 🌿 🌻 🍉';
+  } else if (tempValue >= 70 && tempValue < 80) {
+    state.tempColor = 'orange';
+    state.landscape = '🌿 🌻 🐛 🍀 🌼 🌿 🦋 🍄 🌺 🐝';
+  } else if (tempValue >= 60 && tempValue < 70) {
+    state.tempColor = 'gold';
+    state.landscape = '🌸 🐝 🌱 🐿️ 🌷 🐞 🍃 🐌 🪻 🐇';
+  } else if (tempValue >= 50 && tempValue < 60) {
+    state.tempColor = 'green';
+    state.landscape = '🛷 🍂 ☃️ 🍁 🌲 🍂 🪵 🍁 ⛄️ 🌲 🍂';
+  } else if (tempValue < 50) {
+    state.tempColor = 'teal';
+    state.landscape = '🛷 🍂 ☃️ 🍁 🌲 🍂 🪵 🍁 ⛄️ 🌲 🍂';
+  }
+};
+
+const handleSky = () => {
+  let skySelect = state.skySelect.value;
+  if (skySelect === 'sunny') { 
+    state.sky.textContent = '☀️ 😎 ☀️ 🫠 ☀️ 😎 ☀️ 🫠 ☀️ 😎 ☀️ 🫠 ☀️'; 
+    state.gardenContent.classList = 'garden__content sunny';
+  } else if (skySelect === 'cloudy') {
+    state.sky.textContent = '🌥️ 😶‍🌫️ ⛅️ ☁️ 🌥️ 😶‍🌫️ ⛅️ ☁️ 🌥️ 😶‍🌫️ ⛅️ ☁️'; 
+    state.gardenContent.classList = 'garden__content cloudy';
+  } else if (skySelect === 'rainy') {
+    state.sky.textContent = '🌦 🌈 ⚡️ 🌧 ⛈ 🌈 ⚡️ 🌩️ 🌦 🌈 ⚡️ 🌧'; 
+    state.gardenContent.classList = 'garden__content rainy';
+  } else if (skySelect === 'snowy') {
+    state.sky.textContent = '🌨 ❄️ 🥶 ❄️ 🌨 ❄️ 🥶 ❄️ 🌨 ❄️ 🥶 ❄️ 🌨'; 
+    state.gardenContent.classList = 'garden__content snowy';
+  } else if (skySelect === 'catsAndDogs') {
+    state.sky.textContent = '🐱 🐈‍⬛ 🐶 🐩 🐱 🐈 🐶 🐕‍🦺 🐱 🐈 🐶 🦮'; 
+    state.gardenContent.classList = 'garden__content catsAndDogs';
+  } else if (skySelect === 'chooseSky') {
+    state.sky.textContent = '✨ 🌙 ✨ 🌞 ✨ 🌖 ✨ 🪐 ✨ 🌗 ✨ 💫'; 
+    state.gardenContent.classList = 'garden__content chooseSky';
+  }
+};
+
+const handleTempUpdate = () => {
+  handleTempColorAndGarden(state.tempValue)
+  document.getElementById('tempValue').style.color = state.tempColor;
+  document.getElementById('tempValue').innerHTML = state.tempValue;
+  document.getElementById('landscape').innerHTML = state.landscape;
+};
+
+const handleTempBtnClick = (direction) => {
+  if (direction === 'up') {
+    state.tempValue += 1; 
+  }
+  else if (direction === 'down') {
+    state.tempValue -= 1;
+  }
+  handleTempUpdate();
+};
+
+const handleLatLon = () => {
+  axios.get('http://localhost:5000/location',
+  {
+    params: {
+    q: state.cityName
+      }
+  })
+  .then((response) => {
+    state.lat = response.data[0].lat;
```

Comment:
Updating the state object's `lat` and `lon` values here works totally well such that when you call `handleWeather`, it reads those values from state. However, since they're not being used elsewhere, it's not advantageous to store them in state. You could pass them to the `handleWeather()` function as parameters instead!